### DISCUSSION TITLE
[Feature Request]

### DISCUSSION THREAD

**Discussion by arnav56745** [microsoft/monaco-editor#3910] [Q&A] [answered]
### Context

- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*
- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*

### Description

How can I change the language of the editor?
For example:
`let editor = monaco.editor.create(document.getElementById('editor'), {
    value: '',
    language: 'javascript'
})
editor.updateOptions({
   language: 'php'
})`

**Comment by spahnke** [comment, marked as answer]
The editor does not have an associated language, only models have since there could be multiple models that all have different languages associated with them. The language property on the editor creation function is just a shorthand that sets the language of the model the editor automatically creates for you. So to change a language you need to get the model and then call `monaco.editor.setModelLanguage`:
https://microsoft.github.io/monaco-editor/playground.html?source=v0.37.1#XQAAAALOAQAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscw0BRoNohO0pA3UnSQetsfyL9Rw59E2COVxnxnvCRAg0kwCdQb8ogwu-POzKkkdtD5T7ftMOciItf5sNfdz3glkVyhab3fb-vdOYqBGSKPB_xWM4Wo2Qm9AVrnpr3IpkaLL4w8STWIndQuehaZZLzSPXLcjAkbrNOQeVkwRHWE87KjOFSQIjx5Qlkz4pMkOeRml_6A2u39zNX8rlNvqsv0SnIGiPn3kUGzyeMs8upiIU6zxlVTJm5Zs1DOdRU6kihKafPf_VyYS-XKV-jrxRNZWedWrE0TsoVSGQs_J8KC6bk61p4bKGVuu9i6qBcd8jZpfA9YkDbb5tTAP2ZUoQOj_ggvK54IMPK6r6neesjuC9Igb__vmXpQ
